src/studio/src/designer/backend/Controllers/RepositorySettingsController.cs

```diff
@@ -0,0 +1,77 @@
+using System;
+using System.IO;
+using System.Net;
+using System.Threading.Tasks;
+using Altinn.Studio.Designer.Helpers;
+using Altinn.Studio.Designer.Models;
+using Altinn.Studio.Designer.Services.Interfaces;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Altinn.Studio.Designer.Controllers
+{
+    /// <summary>
+    /// Controller containing actions to handle repository settings aka Altinn Studio settings.
+    /// </summary>
+    ///     //[AutoValidateAntiforgeryToken]
```

Comment:
Should this be commented back in or deleted?

Reply:
Forgot, commented back.